.github/workflows/driver_test_vm.yml

```diff
@@ -32,29 +33,69 @@ jobs:
       with:
         submodules: 'recursive'
 
-    - name: Restore NuGet packages
+    - name: Add MSBuild to PATH
+      uses: microsoft/setup-msbuild@v1
+
+    - name: Install LLVM and Clang
       working-directory: ${{env.GITHUB_WORKSPACE}}
-      shell: cmd
       run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat"
-        nuget restore ${{env.SOLUTION_FILE_PATH}}
+        curl -fsSL -o LLVM10.exe https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe
+        7z x LLVM10.exe -y -o"C:/Program Files/LLVM"
+        echo "C:\Program Files\LLVM\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+    - name: Restore NuGet packages
+      working-directory: ${{env.GITHUB_WORKSPACE}}
+      run: nuget restore ${{env.SOLUTION_FILE_PATH}}
 
     - name: Create verifier project
       working-directory: ${{env.GITHUB_WORKSPACE}}
-      shell: cmd
       run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat"
         cd external\ebpf-verifier
-        rmdir /s /q build
         mkdir build
         cmake -B build
 
     - name: Build
       working-directory: ${{env.GITHUB_WORKSPACE}}
-      shell: cmd
-      run: |
-        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat"
-        msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} ${{env.SOLUTION_FILE_PATH}}
+      run: msbuild /m /p:Configuration=${{env.BUILD_CONFIGURATION}} /p:Platform=${{env.BUILD_PLATFORM}} ${{env.SOLUTION_FILE_PATH}}
+
+    - name: Upload Build Output
+      uses: actions/upload-artifact@v2.2.4
+      with:
+        name: Build x64 ${{ matrix.configurations }}
+        path: ${{ github.workspace }}/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}
+        retention-days: 5
+
+  run_tests:
+    strategy:
+      matrix:
+        configurations: [Debug, Release]
+    runs-on: [windows, ebpf_cicd_tests]
+    env:
+      # Configuration type to build.
+      BUILD_CONFIGURATION: ${{matrix.configurations}}
+
+      BUILD_PLATFORM: x64
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.workflow_run.head_branch }}
+
+    - name: Wait for build to succeed
+      uses: fountainhead/action-wait-for-check@v1.0.0
+      id: wait-for-build
+      with:
+        timeoutSeconds: 900
+        intervalSeconds: 60
```

Comment:
does this mean it might take up to 60 seconds after the build completes before it will start running the tests?  If so, we should probably lower this number to more like 15